assets/react-hooks/probe-styles.js

```diff
@@ -0,0 +1,129 @@
+import { useState, useEffect, useCallback } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { mapValues, keyBy } from 'lodash';
+
+import { hexToRGB } from '../shared/helpers/colors';
+
+const { getComputedStyle } = window;
+
+/**
+ * Get color slugs by probe.
+ *
+ * @return {Object} Object containing the color slugs, where the key is the probe key.
+ */
+const useColorSlugsByProbe = () => {
```

Comment:
I tried to get rid of this hook, just setting the color directly, but I couldn't.

In some cases, it won't recognize the respective theme color because we can have `#fff` or `#ffffff`, for example. So we need this comparison with sanitization to make sure we're selecting the respective theme color.